### ISSUE TITLE
Nama not setup

### ISSUE THREAD

**Issue by mohanarchu** [AgoraIO/FaceUnity#52]
Getting nama not setup error

**Comment by yoreland** [comment]
if you are using iOS, did you get successful result from 'pod install'

**Comment by mohanarchu** [comment]
Yes, getting successful results and FUCamara is also working fine

**Comment by yoreland** [comment]
did you fetch an authKey from faceUnity side?

**Comment by mohanarchu** [comment]
Yes, I have it. When I use in example project it works fine, while using in my own projects then the issue is happening

**Comment by mohanarchu** [comment]
Error is not coming now, But the filters are not working

**Comment by mohanarchu** [comment]
Authentication error #99f9: invalid certificate faceunity

**Comment by yoreland** [comment]
this issue caused from FaceUnity side

**Comment by mohanarchu** [comment]
How to resolve that ? But in example project it was working fine
